src/components/admin/article/ArticleController.jsx

```diff
@@ -12,7 +12,7 @@ import LoadingOverlay from 'components/admin/LoadingOverlay';
 import SaveButton from 'components/admin/article/components/SaveButton';
 import UnpublishButton from 'components/admin/article/components/UnpublishButton';
 import ImageUrlField from 'components/admin/article/components/ImageUrlField';
-import TitleField from 'components/admin/article/components/TitleField';
+import { BoundRequiredTextField } from 'components/admin/form-components/BoundRequiredTextField';
```

Comment:
I think bounded is a better naming here? Bound makes me think of either rope or `function.bind(this)`

Reply:
It's named that because its basically a RequiredTextField, but with an `onChange` bound to `this`.